devices/src/legacy/rtc_pl031.rs

```diff
@@ -107,8 +109,8 @@ impl RTC {
             }
             RTCLR => {
                 self.load = val;
-                self.previous_now = SteadyTime::now();
-                self.tick_offset = Timespec::new(i64::from(val), 0);
+                self.previous_now = Instant::now();
+                self.tick_offset = fc_util::seconds_to_nanoseconds(i64::from(val)).unwrap();
```

Comment:
Add a comment about why this unwrap is safe. Or specifically when this unwrap is unsafe i.e. when you have a malicious guest in which case you want to terminate the execution of firecracker.